### ISSUE TITLE
Update "az vm", "az vmss", and "az sig" to include new cmdlets

### ISSUE THREAD

**Issue by D1v38om83r** [Azure/azure-cli#19573]
**Resource Provider**
Microsoft.Compute

**Description of Feature or Work Requested**
https://microsoft.sharepoint.com/:t:/t/ComputeVM/EQnRBJh6aftLkAyE8WT-fXgBy-iXXVYyHa78KyJF9TblrQ?e=DQjBIP

**Minimum API Version Required**
2021-07-01

**Swagger Link**
https://github.com/Azure/azure-rest-api-specs/pull/11318
https://github.com/Azure/azure-rest-api-specs/pull/15061

**Target Date**
2020-10-01

**Comment by yonzhan** [comment]
Compute

**Comment by zhoxing-ms** [comment]
@D1v38om83r  Since we have planed other projects for this sprint, can we develop and release this requirement in next sprint (for ignite, on **11-02**)?

**Comment by D1v38om83r** [comment]
Yes, please. Ignite sprint is fine.

**Comment by zhoxing-ms** [comment]
@D1v38om83r Hi, due to the large workload of this requirement and we are currently working on several projects, our development time may not be sufficient. Could we consider splitting it into several smaller requirements and completing it in batches?
For example, split `sig` related commands and `vm/vmss` related commands into two requirements respectively, so that the higher priority sub requirements can be released in ignite (**11-02**) and the other requirements can be released in the next sprint after ignite. Do you think this plan is feasible?

**Comment by D1v38om83r** [comment]
No, we cannot afford to have any part of requirement be delayed. I could split the requirement if it makes it easier to implement for you, but not for the purpose of delaying a part of it after ignite.

**Comment by zhoxing-ms** [comment]
@D1v38om83r OK, sounds good~ Please help to split it, we will plan to release this requirement in ignite

**Comment by D1v38om83r** [comment]
reduced the scope of this issue to az sig cmdlets.

The changes required for "az vm" and "az vmss" cmdlets are moved to a new issue https://github.com/Azure/azure-cli/issues/19672

**Comment by amjads1** [comment]
@zhoxing-ms - Thanks for the update and prioritizing this requirement for Ignite release. As you know, we are tracking this as a part of Ignite release announcements so can you confirm if this can be released before 11/2 (10/29 would be ideal)?

**Comment by zhoxing-ms** [comment]
@amjads1  We will release it at the official release time on 11/02
